Class-16-(Fetch-&-JSON).md

```diff
@@ -0,0 +1,40 @@
+## In the prep work this week (HTTP and Forms), the students learned:
+
+1. Why we need a standard format for exchanging data.
+1. The rules for the JSON format.
+1. What an API is.
+1. How to use ``fetch`` to request data from an API.
+1. How to make use of the data returned from an API.
+1. The difference between synchronous and asynchronous processes.
+
+## Large Group Time (Instructor Notes)
+
+### Announcements
+
+1. Graded assignment #5 is due before the next class.
+1. Check with your class Candidate Engagement Manager for any other announcements.
+
+### Topics That Require More Attention
+
+1. Restate what "API" stands for, and provide examples of common APIs.
+    1. Why are APIs a good thing?
+    1. What sorts of information do APIs supply?
+    1. How hard is it to create an API?
+1. JSON looks a lot like JavaScript objects, but note the differences.
+1. How is ``fetch`` similar to and different from HTTP requests (GET, POST)?
+1. Live coding. Set up a ``fetch`` example and discuss:
+    1. Syntax,
+    1. How to use the dev tools to view what ``fetch`` does,
+    1. The ``response`` keyword,
+    1. How to use the response from an API to modify the HTML on a webpage.
+1. Studio intro.
+
+## Studio (TA Notes)
+
+1. Forking and cloning a GitHub repository should be old hat by now. If a student requires help, point them to a book example that shows how to fork and clone rather than walking them through the process again.
+1. Absolutely, positively, NO EXCUSES, complete this studio on your own before class.
```

Comment:
"NO EXCUSES! Complete ..."

Reply:
I was wondering about the punctuation here.  Thanks for the assist!

"That that is is that that is not is not"